cypress.config.js

```diff
@@ -2,6 +2,8 @@ const { defineConfig } = require("cypress");
 const browserify = require('@cypress/browserify-preprocessor')
 
 module.exports = defineConfig({
+	projectId: 'nc_photos_cy',
```

Comment:
Should be from the cypress website 
https://dashboard.cypress.io/projects/okzqgr/runs

```suggestion
	projectId: 'okzqgr',
```

Reply:
I just added the CYPRESS_RECORD_KEY secret